x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.ts

```diff
@@ -81,9 +81,10 @@ export const getPolicyListHandler = function (
     }${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.package.name: endpoint`;
     try {
       const listResponse = await fleetServices.packagePolicy.list(soClient, {
-        ...request.query,
+        page: request.query.page,
```

Comment:
are you sure `request.query` here might not contain other fields that otherwise would be expanded in the previous code? (other than `page` and `sortOrder`) ?